src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__review-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  .stars {
+    display: flex;
+    align-items: center;
+
+    &--4 {
+      display: flex;
+    }
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-position: center;
+      background-repeat: no-repeat;
+      background-image: url(../images/star.svg); // Inactive star
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), multiple background properties should be combined into a single shorthand property `background`. Please combine `background-position`, `background-repeat`, and `background-image` into one `background` property.